modules/ml/src/main/java/org/apache/ignite/ml/regressions/logistic/LogisticRegressionSGDTrainer.java

```diff
@@ -157,8 +157,8 @@
      * @param maxIterations The parameter value.
      * @return Model with new max number of iterations before convergence parameter value.
      */
-    public LogisticRegressionSGDTrainer withMaxIterations(int maxIterations) {
-        this.maxIterations = maxIterations;
+    public LogisticRegressionSGDTrainer withMaxIterations(double maxIterations) {
```

Comment:
I understand why you rewrote signature in a such way but it looks a little bit weird for user. Maybe you can determine simple type during parameters enumeration?